install_functions/installVariables.sh

```diff
@@ -7,11 +7,18 @@
 # Foundation.
 
 ## Package vars
-readonly wazuh_major="4.10"
-readonly wazuh_version="4.10.0"
+readonly wazuh_major="4.9"
+readonly wazuh_version="4.9.0"
```

Comment:
The version of both variables must be 4.10 if this PR is for the `4.10.0` branch

Reply:
Done